gobblin-modules/google-ingestion/src/main/java/gobblin/ingestion/google/webmaster/GoogleWebMasterSourceDaily.java

```diff
@@ -45,7 +46,7 @@
   @Override
   GoogleWebmasterExtractor createExtractor(WorkUnitState state, Map<String, Integer> columnPositionMap,
       List<GoogleWebmasterFilter.Dimension> requestedDimensions,
-      List<GoogleWebmasterDataFetcher.Metric> requestedMetrics)
+      List<GoogleWebmasterDataFetcher.Metric> requestedMetrics, JsonArray schemaJson)
```

Comment:
I see that you have changes some contracts (constructors), are the interfaces / classes in these connectors still evolving. If so, we should mark the classes here with @ Alpha annotation